zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -200,6 +206,7 @@ func (ob *Observer) WatchInboundTracker() {
 }
 
 // ProcessInboundTrackers processes inbound trackers
+// TODO(revamp): move inbound tracker logic in a specific file
```

Comment:
Current `inbound.go` contains methods for inbound ticker, inbound tracker, inbound observation. Once ticker gets moved out to `tickers.go`, do you still want to split `inbound.go` into two files: `inbound.go` and `inbound_tracker.go`?

Reply:
Idea is for tickers.go to keep the minimal logic possible, just containing the ticker and calling methods for other file .

Also I think having many files is not an issue if it helps with organization.

Finally, I might think these watcher methods can be put in common for all chain clients, is it something you looked into with the base client refactoring?